src/rdkafka_conf.c

```diff
@@ -1013,6 +1013,38 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           "the client's OAUTHBEARER token.",
           _UNSUPPORTED_OAUTHBEARER
         },
+        { _RK_GLOBAL, "sasl.oauthbearer.method", _RK_C_STR,
+          _RK(sasl.oauthbearer.method),
+          "Enable the functionality of the OAuth/OIDC login callback handler.",
+          _UNSUPPORTED_OAUTHBEARER
+        },
+        { _RK_GLOBAL, "sasl.oauthbearer.client.id", _RK_C_STR,
+          _RK(sasl.oauthbearer.client_id),
+          "A unique string representing the registration information provided "
```

Comment:
Needs to say to what this is provided. Is it the OIDC endpoint?

Reply:
Should also mention that this is only used when sasl.oauthbearer.method is set to oidc.